xla/service/gpu/fusions/transpose_test.cc

```diff
@@ -80,7 +80,7 @@ TEST_F(TransposeTest, ThreadIndexing021) {
         (d0, d1, d2, d3, d4, d5)[s0, s1, s2] -> (
           d3 floordiv 2,
           d0 floordiv 32 + s1 * 4,
-          (d3 mod 2) * 32 + d0 mod 32
+          ### (d3 mod 2) * 32 + d0 mod 32 ###
```

Comment:
What's this?

Reply:
See https://github.com/openxla/xla/commit/a70d147e4e5dcb3824af1aaec91d5b070ae1389c which added this kind of matching

Reply:
That's weird. We should probably fix that properly before it spreads everywhere. These expressions must have a canonical form.